src/Microsoft.DotNet.Wpf/src/DirectWriteForwarder/CPP/DWriteWrapper/Factory.cpp

```diff
@@ -92,7 +92,9 @@ namespace MS { namespace Internal { namespace Text { namespace TextInterface
         _pFactory = (IDWriteFactory*)factoryTemp;
     }
 
+    #pragma warning (disable : 4950) // The Constrained Execution Region (CER) feature is not supported.  
```

Comment:
If CER is not supported, then how do you plan on ensuring that this code works as intended? Simply suppressing the warning won't do that, would it?

/cc @SamBent

Reply:
https://github.com/dotnet/designs/blob/master/accepted/2020/better-obsoletion/obsoletions-in-net5.md

The CER API calls need to be removed before GA; this is intended to unblock the builds.